src/objects/SkinnedMesh.js

```diff
@@ -159,21 +179,21 @@ SkinnedMesh.prototype = Object.assign( Object.create( Mesh.prototype ), {
 
 	},
 
-	updateMatrixWorld: function () {
+	updateMatrixWorld: function ( force ) {
 
-		Mesh.prototype.updateMatrixWorld.call( this, true );
```

Comment:
Not 100% sure, but this looks redundant to me. I can see no problems as long as we correctly pass the `force` parameter to `.updateMatrixWorld()`.